types/spinnies/index.d.ts

```diff
@@ -0,0 +1,150 @@
+// Type definitions for spinnies 0.5
+// Project: https://github.com/jcarpanelli/spinnies
+// Definitions by:  Adam Jarret <https://github.com/adamjarret>
+//                  Benjamin Burns <https://github.com/benjamincburns>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace Spinnies;
+export = Spinnies;
+
+declare namespace Spinnies {
+    type StopAllStatus = "succeed" | "fail" | "stopped";
+    type SpinnerStatus = StopAllStatus | "spinning" | "non-spinnable";
+
+    interface Spinner {
+        interval: number;
+        frames: string[];
+    }
+
+    interface SpinnerOptions {
+        /**
+         * Optional text to show in the spinner. If none is provided, the name field will be shown.
+         */
+        text?: string;
+
+        /**
+         *  Optional, indent the spinner with the given number of spaces.
+         */
+        indent?: number;
+
+        /**
+         * Initial status of the spinner. Valid statuses are: succeed, fail, spinning, non-spinnable and stopped.
+         */
+        status?: SpinnerStatus;
+
+        /**
+         * Any valid chalk color.
+         */
+        color?: string;
+
+        /**
+         * Any valid chalk color.
+         */
+        succeedColor?: string;
+
+        /**
+         * Any valid chalk color.
+         */
+        failColor?: string;
+
+        /**
+         * Any valid chalk color. The default value is greenBright.
+         */
+        spinnerColor?: string;
+    }
+
+    interface Options {
+        /**
+         * Any valid chalk color. The default value is white.
+         */
+        color?: string;
+
+        /**
+         * Any valid chalk color. The default value is green.
+         */
+        succeedColor?: string;
+
+        /**
+         * Any valid chalk color. The default value is red.
+         */
+        failColor?: string;
+
+        /**
+         * Any valid chalk color. The default value is greenBright.
+         */
+        spinnerColor?: string;
+
+        /**
+         * The default value is ✓.
+         */
+        succeedPrefix?: string;
+
+        /**
+         * The default value is ✖.
+         */
+        failPrefix?: string;
+
+        /**
+         * Disable spins (will still print raw messages).
+         */
+        disableSpins?: boolean;
+
+        /**
+         * Spinner configuration
+         */
+        spinner?: Spinner;
+    }
+}
+
+declare class Spinnies {
+    static dots: Spinnies.Spinner;
+    static dashes: Spinnies.Spinner;
```

Comment:
those are technically not a static symbols of the class, but re-exported symbols on the module level, so:

```ts
declare namespace Spinnies {
    const  dots: Spinner;
    const dashes: Spinner;
    type StopAllStatus = "succeed" | "fail" | "stopped";
    ...
```
```ts
import Spinnies = require("spinnies");

Spinnies.dots; // $ExpectType Spinner
Spinnies.dashes; // $ExpectType Spinner
```